recipes/mo-catnip/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mo-catnip" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 491e72214f2018dcd57e580956c97eb91b6df1cd6172b85a7e9e0da985733e54
```

Comment:
The error from CI indicates that this hash is incorrect. You can actually get the appropriate has from PyPI itself.

Reply:
Thanks, Updated. 

I am not sure if import catnip is correct. Can you please check? I now changed it to lib/catnip 
https://github.com/MetOffice/CATNIP